### PR TITLE
Add automated language testing. Only one pdf downloaded for all.

### PR DESCRIPTION
**Pull request by plocket**
Addresses #37. Note: The package.json script is to allow local developers to test the languages. I think we should keep the non-multi-lingual tests out of github automated runs for now and they could cause tests to go for quite a long while. With that in mind, maybe the stretch goal should is a more immediate need to be brought along pretty quickly.

### CODE REVIEW

**Review by niharikasingh — APPROVED** [review]

File: lib/utils/langs.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env node
+
+let _path = require('path');
+let fs = require('fs');
+let glob = require('glob');
+
+let env_vars = require('./env_vars');
+
+// Get the glob filepaths to find all the test files
+let dir_root = process.env.INIT_CWD;  // Path to root of project
```
[the remark this answers is not in the document]
Comment (plocket) [reply lib/utils/langs.js:10]: See issue #2!

[the remark this answers is not in the document]
Comment (plocket) [reply lib/utils/langs.js:10]: There should also be work on the README so people don't have to open that file to see what they need to do. That's an issue as well. #49

File: lib/utils/env_vars.js [hunk 2]
```diff
@@ -29,5 +29,11 @@ if ( process.env.BRANCH_PATH ) {
 env_vars.PROJECT_NAME = ('testing' + env_vars.BRANCH_NAME).replace(/[^A-Za-z0-9]/gi, '');
 env_vars.BASE_INTERVIEW_URL = `${env_vars.BASE_URL}/interview?new_session=1&i=docassemble.playground${env_vars.PLAYGROUND_ID}${env_vars.PROJECT_NAME}%3A`;
 
+if ( process.env.LANGUAGES ) {
+  env_vars.LANGS = process.env.LANGUAGES.split(', ');
```
[the remark this answers is not in the document]
Comment (plocket) [reply lib/utils/env_vars.js:33]: I should specify the format, though that's not the format. It's actually whatever words are on the button or link for that language. I haven't figured out another way to do it.

[the remark this answers is not in the document]
Comment (plocket) [reply lib/utils/env_vars.js:33]: Also something that should be specified in the README. This is a draft and I'll try to add this note as I develop it further - trying to work out issues with the manual workflow.

File: lib/steps.js [hunk 3]
```diff
@@ -115,6 +115,15 @@ Given(/I start the interview at "([^"]+)"[ on ]?(.*)/, async (file_name, optiona
 
   scope.activate = click_with[ scope.device ];
 
+  // Tap the language button if given
+  let lang_url = null;
+  if ( language ) {
+    let [lang_link] = await scope.page.$x(`//a[text()="${ language }"]`);
+    expect( lang_link, `Could not find the link for the language "${ language }"` ).to.exist;
+    await scope.tapElement( scope, lang_link );
+    await scope.page.screenshot({ path: './screenshots_langauge.jpg', type: 'jpeg', fullPage: true });
```
[the remark this answers is not in the document]
Comment (plocket) [reply lib/steps.js:124]: Oh, that's not supposed to be there anymore. I was just making sure things were happening appropriately. Good catch.

**Comment by plocket** [comment]
0.4.64 will have to be published after this is merged in

**Review by niharikasingh — APPROVED** [review]

**Pull request merged**